### ISSUE TITLE
The Admin user should not be a normal user

### ISSUE THREAD

**Issue by celprov** [nipreps/qkay#24]
IMHO, it makes more sense that the admin user is used only for assigning datasets.
As such, it should not be possible to assign datasets to rate to the admin. 
Additionally, the admin user should not have a page showing him what datasets are assigned to him; it should only have access to the admin panel page.

**Comment by esavary** [comment]
It is mainly intended to facilitate testing during code development, so we don't have to log in and out as both user and admin to test the different panels. However, I have no problem removing it after addressing other issues if it's no longer useful.

**Comment by celprov** [comment]
Yes indeed I experienced it as a handy feature to have for development.
